tcadmin/diff.py

```diff
@@ -95,7 +95,7 @@ def contextualize(rangeInfo):
                 return match.group(1)
         return ""
 
-    lines = difflib.unified_diff(
+    lines = patiencediff.unified_diff(
```

Comment:
Of note, for fxci we've forked slightly to be speedier here:

https://hg.mozilla.org/ci/ci-admin/rev/689bb1ade93bee51966d82268cc7d0148ad66e79

(Though there were some minor bustage fixes, full current version at https://hg.mozilla.org/ci/ci-admin/file/tip/src/tcadmin/diff.py )

Not sure if that is a more desirable solution here or not though.